src/lib/features/export-import-toggles/export-import-service.ts

```diff
@@ -975,17 +969,7 @@ export default class ExportImportService
             dependencies: dependencies.map((d) => d.dependency),
         }));
 
-        const groupedFeatureLinks = groupBy(featureLinks, 'feature');
-        const mappedFeatureLinks = Object.entries(groupedFeatureLinks).map(
-            ([feature, links]) => ({
-                feature,
-                links: links.map((link) => ({
-                    id: link.id,
-                    url: link.url,
-                    title: link.title,
-                })),
-            }),
-        );
+        const mappedFeatureLinks: any[] = [];
```

Comment:
delete this variable. It's not useful. In addition: remove the `links` property of the object that you return from this function (where you use this variable).